theory/energetic sources/AGNs.md

```diff
@@ -7,3 +7,35 @@ nav_order: 1
 ---
 
 ## Active Galactic Nuclei (AGNs)
+
+### Introduction
+
+Active Galactic Nuclei or AGNs as they are known, are very powerful and bright galactic centers, powered by supermassive [black holes](../special%20stars/black%20holes.html). You can think of these black holes as cosmic vacuum cleaners of enormous size, pulling the surrounding gas and dust toward themselves.
+
+This material, as it approaches the black hole, begins to heat up due to friction and then radiates light, emitting energy across the whole spectrum-from radio waves to X-rays. This is the reason why AGNs are so bright; sometimes they outshine all the stars in their host galaxy put together. They are mostly situated at the center of the host galaxy.
+
+### Types of AGNs
+
+Varieties of [AGNs](#active-galactic-nuclei-agns) exist, and their appearance also depends on how we're looking at them. For example, quasars are AGNs that are extremely bright and can be seen from billions of light-years away. Not all AGNs are [quasars](../energetic%20sources/quasars.html). Some [quasars](../energetic%20sources/quasars.html) are so far away that the light we see today started its journey when the universe was still young.
```

Comment:
Please hyperlink the second AGN to the title like to did for the first one. Similar comments for the other files in the PR

Reply:
Resolved now